$NetBSD: patch-mosquitto.conf,v 1.9 2026/01/30 01:07:56 gdt Exp $

Align pid_file to pkgsrc norms.

Log to syslog, instead of (perhaps) not logging.

--- mosquitto.conf.orig	2025-03-06 16:25:31.000000000 +0000
+++ mosquitto.conf
@@ -167,7 +167,7 @@
 # This should be set to /var/run/mosquitto/mosquitto.pid if mosquitto is
 # being run automatically on boot with an init script and
 # start-stop-daemon or similar.
-#pid_file
+pid_file @VARBASE@/run/mosquitto/mosquitto.pid
 
 # Set to true to queue messages with QoS 0 when a persistent client is
 # disconnected. These messages are included in the limit imposed by
@@ -469,7 +469,7 @@
 # Note that if the broker is running as a Windows service it will default to
 # "log_dest none" and neither stdout nor stderr logging is available.
 # Use "log_dest none" if you wish to disable logging.
-#log_dest stderr
+log_dest syslog
 
 # Types of messages to log. Use multiple log_type lines for logging
 # multiple types of messages.
